snooty/postprocess.py

```diff
@@ -903,6 +1061,45 @@ def build_slug_fileid_mapping(self) -> None:
             fileid_dict[slug] = fileid
         self.slug_fileid_mapping = fileid_dict
 
+    def build_iatree(self) -> Dict[str, SerializableType]:
+        starting_page = self.pages.get(FileId("index.txt"))
+
+        if not starting_page:
+            return {}
+        if not isinstance(starting_page.ast, n.Root):
+            return {}
+        if not starting_page.ast.options.get("ia"):
```

Comment:
`"ia" not in starting_page.ast.options` might make the intent a little more clear